articles/active-directory-b2c/openid-connect.md

```diff
@@ -31,31 +31,31 @@ Azure AD B2C extends the standard OpenID Connect protocol to do more than simple
 
 When your web application needs to authenticate the user and run a user flow, it can direct the user to the `/authorize` endpoint. The user takes action depending on the user flow.
 
-In this request, the client indicates the permissions that it needs to acquire from the user in the `scope` parameter, and specifies the user flow to run. To get a feel for how the request works, try pasting the request into a browser and running it. Replace `{tenant}` with the name of your tenant. Replace `90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6` with the app ID of the application you've previously registered in your tenant. Also change the policy name (`{policy}`) to the policy name that you have in your tenant, for example `b2c_1_sign_in`.
+In this request, the client indicates the permissions that it needs to acquire from the user in the `scope` parameter, and specifies the user flow to run. To get a feel for how the request works, try pasting the request into a browser and running it. Replace `{tenant}` with the name of your tenant. Replace `90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6` with the app ID of an application you've previously registered in your tenant or [register a web application](tutorial-register-applications.md#register-a-web-application). Replace `{application-id-uri}/{scope-name}` with the **Application ID URI** and **Scope** of an application that you've previously registered in your tenant. Go to the Azure AD B2C blade, select **App registrations**, select **All applications** and select an application. Under the **Manage** section, select **Expose an API**. If it's preferable to start fresh, [register a new web api](add-web-api-application.md#app-registrations) and [configure scopes](add-web-api-application.md#configure-scopes). Also change the policy name (`{policy}`) to the policy name that you have in your tenant, for example `b2c_1_sign_in`.
 
 ```http
 GET https://{tenant}.b2clogin.com/{tenant}.onmicrosoft.com/{policy}/oauth2/v2.0/authorize?
 client_id=90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6
-&response_type=code+id_token
-&redirect_uri=https%3A%2F%2Faadb2cplayground.azurewebsites.net%2F
+&response_type=code+token
+&redirect_uri=https%3A%2F%2Fjwt.ms%2F
 &response_mode=fragment
-&scope=openid%20offline_access
+&scope=openid%20offline_access%20{application-id-uri}/{scope-name}
```

Comment:
```suggestion
&scope=openid%20offline_access
```

Reply:
at this point, we're interested in id_token to identify a user, we don't need to send any other scope other than offline_access and openid.